src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_csharp_cli.py

```diff
@@ -144,3 +144,21 @@ def mock_input(*args, **kwargs):
 
     def test_flow_run_from_resume(self):
         run_pf_command("run", "create", "--resume-from", "net6_0_variant_0_20240326_163600_356909")
+
+    def test_flow_class_init(self):
+        """Note that this test won't pass. Instead, it will hang and pop up a web page for user input.
+        Leave it here for debugging purpose.
+        """
```

Comment:
Should we add some mock apis here and assert the method is called?
As you can see you have make so much layers in executor proxy.

Add some mocks to avoid C# is broken.

Reply:
Got an idea that we can trigger a ado pipeline from github like current compliance check. will do this in a separate PR